bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/flamingos2/Flamingos2.java

```diff
@@ -852,8 +856,25 @@ public static double getFpuChangeOverheadSec() {
         return CommonStepCalculator.instance.calc(cur, prev).addAll(times);
     }
 
+    private static final double IMAGING_SETUP_TIME_OIWFS = 60 * 15;
+
+    private static final double IMAGING_SETUP_TIME_PWFS2 = 60 * 6;
```

Comment:
I realize these are (a) trivial calculations that are immediately obvious, and (b) perfectly consistent with how these things are defined all over the OCS codebase.  Still, the new time API in Java offers a slight improvement here that we have lately been taking advantage of:

```java
    import java.time.Duration
    ...
    private static final Duration IMAGING_SETUP_TIME_OIWFS = Duration.ofMinutes(15);
    private static final Duration IMAGING_SETUP_TIME_PWFS2 = Duration.ofMinutes(6);
    ...
    public static double getImagingSetupSec(ISPObservation obs) {
        return (usesF2Oiwfs(obs) ? IMAGING_SETUP_TIME_OIWFS : IMAGING_SETUP_TIME_PWFS2).getSeconds();
    }
```

Really the `getImagingSetupSec` should return a `Duration` but that would be a bigger refactoring that we can skip since we should do this better in the new model.